packages/connect-examples/new-example/app/components/ui/Checkbox.tsx

```
@@ -0,0 +1,30 @@
+"use client"
```

ESLint

[error] Resolve error: typescript with invalid interface loaded as resolver

(import/namespace)